packages/language-server-tests/vitest.config.ts

```diff
@@ -0,0 +1,11 @@
+import { defineConfig } from "vitest/config";
+
+export default defineConfig({
+	test: {
+		dir: "tests",
+		// Both settings are needed so that tests can share the same language server
+		isolate: false,
+		fileParallelism: false,
```

Comment:
I wish we didn't need that one, but I couldn't figure it out. Maybe it'd need to work completely differently, like they all connect to a running process defined elsewhere, no idea